typescript/packages/core/package.json

```diff
@@ -12,6 +12,9 @@
     "module": "./dist/index.mjs",
     "types": "./dist/index.d.ts",
     "dependencies": {
+        "@metaplex-foundation/mpl-bubblegum": "^4.2.1",
+        "@metaplex-foundation/umi-bundle-defaults": "^0.9.2",
+        "@metaplex-foundation/umi-web3js-adapters": "^0.9.2",
```

Comment:
could this be in a plugin specific directory, to make the core thinner for everyone not interested?

Reply:
The core already has a bunch of solana and evm-specific stuff, lets move them to separate packages as a follow-up!

Reply:
hmm the challenge is I think other packages (e.g. eliza) already depend on goat, so adding this here and unless we remove it before another release, we'll increase bloat on anyone downstream and maintainers of those repos may get annoyed

Reply:
I thought we were moving this to its own `nft` plugin.

The specific stuff for core is for the wallet client, everything else should be outside since its for specific use cases. Someone that does DeFi might not care at all about the nft dependencies